package.json

```diff
@@ -34,9 +29,9 @@
     "lint": "mkdir -p build2 && run-p -c lint:eslint lint:grunt lint:stylelint",
     "lint:report": "mkdir -p build2 && run-p -c lint:eslint:report lint:grunt:report lint:stylelint:report",
     "test:jest": "grunt assets && yarn jest",
-    "test:dev": "grunt assets && karma start",
-    "test:coverage": "yarn test --reporters=progress,coverage",
-    "test": "yarn test:dev --single-run",
+    "test:jest:coverage": "yarn test:jest --collectCoverage",
```

Comment:
Adding an extra script to capture coverage seems unnecessary. Instead, what if we took the `okta-ui` approach and added a README note about passing through `jest` options?

Reply:
Yeah, we could update README.